package-lock.json

```diff
@@ -0,0 +1,2086 @@
+{
```

Comment:
The `package-lock.json` file is a npm 5 feature but since [apm still uses npm 3](https://github.com/atom/apm/blob/master/package.json#L35) the `package-lock.json` is not used. You can install an Atom package with npm but then it will use your systems version of node and not Atoms. This in turn will result possible wrong dependencies and incompatibility with native modules.